rwgame/states/DebugState.cpp

```diff
@@ -317,18 +317,22 @@ std::shared_ptr<Menu> DebugState::createMissionsMenu() {
 
             if (vm) {
                 std::list<SCMThread>& threads = vm->getThreads();
+                const auto& offsets = vm->getFile()->getMissionOffsets();
+
+                RW_ASSERT(!offsets.empty());
 
                 for (auto& thread : threads) {
-                    if (thread.isMission) {
+                    if (thread.baseAddress >= offsets[0]) {
```

Comment:
The comment above said that the baseAddress does not exist in III and VC. But now it's used.
I'm confused.

Reply:
Ah, I assume the comment above referred to Savegames only.

Reply:
openrw implements scripts differently, instead of loading mission script bytecode into fixed .bss array, it just loads whole scm and uses baseAddress for mission relative jumps, meanwhile VC for example just uses fixed offset 225512 or for III - 0x20000

Reply:
Is the meaning of the field documented anywhere? It would be good to correct the discrepancy.

Reply:
what field?

Reply:
baseAddress

Reply:
Ah, then I just misunderstood. I thought you meant that it just isn't in the save game?

Reply:
If mission thread needs to do relative jump it subtracts negative jump offset from fixed offset, there is a special buffer for mission bytecode.